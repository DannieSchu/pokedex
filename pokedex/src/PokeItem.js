import React, { Component } from 'react';

export default class PokeItem extends Component{
    render() {
      return (
        <li>
            <img />
            <h1></h1>
            <p></p>
            <p></p>
        </li>
      );
    }
  }
