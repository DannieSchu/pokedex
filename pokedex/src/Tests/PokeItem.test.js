import React from 'react'
import renderer from 'react-test-renderer';
import PokeItem from '../PokeItem';

test('renders PokeItem.js correctly', () => {
    const item = {}
    const tree = renderer
      .create(<PokeItem pokemon= {item} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });