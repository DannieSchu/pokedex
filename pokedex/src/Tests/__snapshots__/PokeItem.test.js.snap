// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders PokeItem.js correctly 1`] = `
<li
  className="card-container"
>
  <img />
  <p>
    Attack: 
    ; Defense: 
  </p>
  <h2 />
</li>
`;
